lib/porky_lib/unencrypted/file_service.rb

```diff
@@ -25,6 +25,7 @@ def read(bucket_name, file_key, options = {})
     tempfile.read
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Usually this is a sign the method needs to be split, try to avoid if possible

Reply:
Ok, I will split it